script.js

```diff
@@ -22,6 +54,31 @@ function clearInputs() {
 
 function deleteIdea(event) {
   if (event.target.classList.contains('delete-btn')) {
```

Comment:
if you do what I suggest above you can eliminate this check since your only going to get clicks on the button and not the entire idea